docroot/themes/custom/twentyseventeen/sass/base/form/_form.scss

```diff
@@ -102,6 +103,30 @@ label.option {
   display: inline-block;
 }
 
+
+.form-file {
+  width: 0.1px;
+  height: 0.1px;
+  opacity: 0;
+  overflow: hidden;
+  position: absolute;
+  z-index: -1;
+}
+
+.js-form-managed-file.form-managed-file input {
+  font-size: 1.2em;
+  font-weight: normal;
+  color: $wht;
+  background: rgba(0,62,106,1) url('../images/GovCon-FileUpload-SM-Rev.svg') 8% center no-repeat;
+  background-size: 1em;
+  padding: 0.25em 0.5em 0.125em 2.5em;
+  border-radius: 0.25em;
+  border: 1px solid rgba(0,62,106,1);
+  display: inline-block;
+}
+
+.js-form-managed-file.form-managed-file input#edit-field-files-0-upload.js-form-file.form-file
```

Comment:
This CSS selector can be simplified.  Right now its keyed to a specific field ID which is pretty brittle. I would recommend to instead target a simpler class like .form-file?